include/nuttx/net/netdev.h

```diff
@@ -161,8 +161,8 @@
  * headers
  */
 
-#define IPBUF(hl) ((FAR void *)\
-                   &dev->d_iob->io_data[CONFIG_NET_LL_GUARDSIZE + (hl)])
+#define IPBUF(hl) ((FAR void *)(IOB_DATA(dev->d_iob) + (hl)))
+#define LLBUF     (IPBUF(-NET_LL_HDRLEN(dev)))
```

Comment:
```suggestion
#define LLBUF     (IPBUF - (NET_LL_HDRLEN(dev)))

Reply:
The LLBUF is too generic, maybe NETLLBUF is better to make it easy to understand its purpose

Reply:
Done.